controllers/get_topics.js

```diff
@@ -22,3 +22,12 @@ exports.getAllArticles = (req, res, next) => {
     .catch(next);
 }
 
+exports.getCommentsById = (req, res, next) => {
+    const {article_id} = req.params
+    Promise.all([selectCommentsById(article_id), checkArticleExists(article_id)])
```

Comment:
Great use of `Promise.all` to check the article exists before getting the comments